### ISSUE TITLE
Missing Cancel button in General Settings page

### ISSUE THREAD

**Issue by tuxpiper** [ushahidi/platform#4737]
Missing cancel button in General settings
![Screenshot 2023-10-18 at 11 32 08](https://github.com/ushahidi/platform/assets/8624777/541c1bcd-5192-456b-8d17-cb5c1060dae4)

---
Created automatically from USH-874 (https://linear.app/ushahidi/issue/USH-874/missing-cancel-button-in-general-settings-page)

**Comment by kivuvarosekivuvan** [comment]
Hello, Can I work on this as an Outreachy applicant 2024

**Comment by goodylove** [comment]
@Angamanga please can i work on this issue i have successful done the necessary setup

**Comment by Angamanga** [comment]
@kivuvarosekivuvan Please go ahead! @goodylove I will try to keep one contributor per issue, could you ping me on another one that you find interesting and I'll assign you!

**Comment by crocmons** [comment]
Is this task still in progress? If not, can you assign me this issue, @Angamanga?  I would love to work on it.

**Comment by Angamanga** [comment]
@crocmons I have seen work being done on this already. @kivuvarosekivuvan How are you getting on? Do you need any guidance? @crocmons I suggest you look at the "open for everyone" issues in the project board for now and try do work. I will ping you if this issue becomes available 👍

**Comment by crocmons** [comment]
@Angamanga yeah sure, thanks

**Comment by kivuvarosekivuvan** [comment]
@Angamanga I raised a PR and I was waiting for review
Here is the PR: [https://github.com/ushahidi/platform-client-mzima/pull/813](url)

**Comment by Angamanga** [comment]
Done
